src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -173,11 +173,23 @@ public StreamingResponseBody streamEvents(
         };
     }
 
+    private String invalidCursorMessage(final CursorError error, final Cursor cursor) {
+        switch (error) {
+            case PARTITION_NOT_FOUND:
+                return "non existing partition " + cursor.getPartition();
+            case EMPTY_PARTITION:
+                return "partition " + cursor.getPartition() + " is empty";
+            case UNAVAILABLE:
+                return "offset " + cursor.getOffset() + " for partition " + cursor.getPartition() + " unavailable";
```

Comment:
Maybe I should move it to closer to the noun, like "unavailable offset for partition".